pkg/ccl/backupccl/backup_planning.go

```diff
@@ -79,6 +80,22 @@ type backupKMSEnv struct {
 
 var _ cloud.KMSEnv = &backupKMSEnv{}
 
+// TODO(angelaw): technically this does not encompass ALL bulk IO features at the moment--
+// We will need documentation somewhere (where is best?) to indicate which features are covered
+// under which categories, and which features can already be enabled/disabled.
+
+// featureBulkIOEnabled is used to enable and disable relevant features under the bulk IO category.
+var featureBulkIOEnabled = settings.RegisterPublicBoolSetting(
+	"feature.bulkio.enabled",
+	"set to true to enable bulk IO features, false to disable; default is true",
+	true /* enabled by default */)
+
+// featureBackupEnabled is used to enable and disable the BACKUP feature.
+var featureBackupEnabled = settings.RegisterPublicBoolSetting(
+	"feature.bulkio.backup.enabled",
+	"set to true to enable backups, false to disable; default is true",
+	true /* enabled by default */)
```

Comment:
nit: this should just be the varname, i.e. `true /* defaultValue */,`

even better if we can define a `const FeatureFlagEnabledDefault = true` somewhere -- not opposed to creating a `pkg/featureflag` for this const (for now)

Reply:
Good idea! Do you ever see a case where that const is set to false? 🤔 Or is the idea of the constant just for clarity?

Reply:
clarity -- see also "avoid naked parameters" in https://wiki.crdb.io/wiki/spaces/CRDB/pages/181371303/Go+coding+guidelines